newswires/app/controllers/ViteController.scala

```diff
@@ -47,10 +50,11 @@ class ViteController(
   }
   private val headersToKeep =
     Seq(CACHE_CONTROL, ETAG, DATE, ACCESS_CONTROL_ALLOW_ORIGIN)
+  private val devEmails = configuration.get[String]("devEmails").split(",")
```

Comment:
I've created an SSM param: `/PROD/editorial-feeds/newswires/devEmails` (and one for CODE)
not in cloudformation, following the pattern of other runtime SSM params for this app